src/picongpu/include/simulationControl/MovingWindow.hpp

```diff
@@ -231,8 +231,8 @@ class MovingWindow
             float_64 offsetFirstGPU = 0.0;
             getCurrentSlideInfo(currentStep, NULL, &offsetFirstGPU);
 
-            /* global offset is all 0 except for y dimension */
-            window.globalDimensions.offset.y() = offsetFirstGPU;
+            /* moving window can only slide in y direction */
+            window.globalDimensions.offset.y() += offsetFirstGPU;
```

Comment:
should stay "=" (see "globalWindow.offset" [definition](https://github.com/ComputationalRadiationPhysics/picongpu/wiki/PIConGPU-domain-definitions))